aptos-move/mvhashmap/src/types.rs

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use aptos_aggregator::delta_change_set::DeltaOp;
```

Comment:
I feel like you should move the executable trait definition to here rather than in `aptos-types`?

Reply:
I have added in types because putting it in aptos-move screams "circular-dependency", as this is a trait that Move-VM has to implement. As you said, maybe we move it to Move-VM in a few PRs, but in the interim this seemed a more logical place due to that reasoning.